index.html

```diff
@@ -26,8 +27,9 @@ <h1>heading1</h1>
             <figure class="mugshot">
                 <!-- <img src="images/gradient.png" alt="THUMBNAIL"> -->
                 <a href="links.html">
-                    <img src="Sault College Logo Transparent.png" width="10%" />
+                    <img src="Sault%20College%20Logo%20Transparent.png" alt="Sault College Logo" />
```

Comment:
Space characters should be escaped in HTML attribute values that contain URLs
Missing alt attribute